lib/vdsm/storage/lvm.py

```diff
@@ -1174,6 +1179,9 @@ def deactivateUnusedLVs(vgname, skiplvs=()):
             elif lv.opened:
                 log.debug("Skipping open lv: vg=%s lv=%s", vgname,
                           lv.name)
+            elif completelvname in keep_active_lvs:
```

Comment:
```suggestion
            elif lvPath(vgname, lv.name) in KEEPACTIVE:
```
And you can save both local variables above.